src/protocols/mp_services.rs

```diff
@@ -0,0 +1,122 @@
+//! MP Services Protocol
+//!
+//! This Protocol is defined in UEFI Platform Integration Specification, Section 13.4
+//!
+//! The MP Services Protocol provides a generalized way of performing following tasks:
+//! - Retrieving information of multi-processor environment and MP-related status of specific processors.
+//! - Dispatching user-provided function to APs.
+//! - Maintain MP-related processor status.
+
+pub const PROTOCOL_GUID: crate::base::Guid = crate::base::Guid::from_fields(
+    0x3fdda605,
+    0xa76e,
+    0x4f46,
+    0xad,
+    0x29,
+    &[0x12, 0xf4, 0x53, 0x1b, 0x3d, 0x08],
+);
+
+pub type StatusFlag = u32;
```

Comment:
I dropped this type. We only add the types if they spec does so (and usually in this case only for enums). But in this case the spec has no enum-type, so I think it is fine to just define the flags. We do that in many other cases (e.g., ICMP_* in ip6.rs).